flexget/plugins/urlrewrite/newpct.py

```diff
@@ -24,14 +24,21 @@ class UrlRewriteNewPCT(object):
     def url_rewritable(self, task, entry):
         url = entry['url']
         rewritable_regex = '^http:\/\/(www.)?newpct1?.com\/.*'
-        return re.match(rewritable_regex, url) and not url.startswith('http://www.newpct.com/descargar/')
+        return re.match(rewritable_regex, url) and not url.startswith('http://www.newpct1.com/download/') and not url.startswith('http://www.newpct.com/torrents/')
 
     # urlrewriter API
     def url_rewrite(self, task, entry):
         entry['url'] = self.parse_download_page(entry['url'])
 
     @plugin.internet(log)
     def parse_download_page(self, url):
+        if 'newpct1' in url:
+            return self.parse_newpct1(url)
+        else:
+            return self.parse_newpct(url)
+
+    def parse_newpct(self, url):
+        log.verbose('Newpct URL: %s' % url)
```

Comment:
`log` is capable of string formatting, so stuff like this should be written as `log.verbose('Newpct URL: %s', url)`